src/config/dark-sites.config

```diff
@@ -42,6 +42,7 @@ akkoma.dev
 albony.xyz
 alexkaehler.com
 alfawal.dev
+alecks.dev
```

Comment:
sites should be sorted alphabeticaly...

this site should be before `alexkaehler.com`

Reply:
Fixed it